### PR TITLE
50 fix initial linter errors

### PR DESCRIPTION
**Pull request by nimitz-Bear**
This pull request adds a linter to the pipeline. The linter also has a `lint.xml` file in the project root that is used to disable some of the more unnecessary linter errors.

**Steps Done**
- [x] created an `android-lint.yml` CI workflow file that handles running the linter on the pipeline
- [x] fixed all of the errors in `main` to ensure that the CI workflow would pass
- [x] Should add a template to pull requests

**Future Additions**
- [ ] create a formatter to ensure that code can be easily changed to comply with the linter

### CODE REVIEW

File: .github/workflows/android-lint.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#      - name: Check lint
+#        run: ./gradlew :app:lintProdRelease :app-nia-catalog:lintRelease :lint:lint
+#
+#      - name: Upload lint reports (HTML)
+#        if: always()
+#        uses: actions/upload-artifact@v3
+#        with:
+#          name: lint-reports
+#          path: '**/build/reports/lint-results-*.html'
```
Comment (silfeh) [inline .github/workflows/android-lint.yml:36]: should this commented section be left in?

File: .github/workflows/android.yml [hunk 2]
```diff
@@ -27,6 +26,10 @@ jobs:
       uses: asadmansr/android-test-report-action@v1.2.0
       if: ${{ always() }} # IMPORTANT: run Android Test Report regardless
 
+#      # Run linter
+#    - name: Linter
+#      run: ./gradlew lint
```
Comment (silfeh) [inline .github/workflows/android.yml:31]: also here

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <resources>
     <string name="app_name">Feel Good Inc</string>
     <!-- TODO: Remove or change this placeholder text -->
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+<!--    <string name="hello_blank_fragment">Hello blank fragment</string>-->
```
Comment (silfeh) [inline app/src/main/res/values/strings.xml:4]: remove this placeholder instead of commenting probably

File: lint.xml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    lintOptions {
+        xmlOutput file("lint-results.xml")
+    }
+    <!-- Disable the IconMissingDensityFolder check in this project -->
+<!--    <issue id="IconMissingDensityFolder" severity="ignore" />-->
+
+    <!-- Ignore the ObsoleteLayoutParam issue in the specified files -->
```
Comment (silfeh) [inline lint.xml:9]: also remove comments here if unneeded

**Review by silfeh — CHANGES_REQUESTED** [review]
some areas that are commented out in the code may need to be looked at.

**Comment by nimitz-Bear** [comment]
@silfeh should be good now

**Review by silfeh — APPROVED** [review]
looks good thank you

**Pull request merged**